x-pack/legacy/plugins/ml/server/models/data_frame/error_utils.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+// @ts-ignore no declaration file for module
+export { isRequestTimeout } from '../job_service/error_utils';
```

Comment:
nit, this function is so small and used quite a few times in this PR, IMO it would be worth adding a definition file for it.

Reply:
Addressing in a follow-up 👍